internal/zinc-benchmarks/src/test/scala/xsbt/ZincBenchmark.scala

```diff
@@ -74,10 +75,10 @@ private[xsbt] class ZincBenchmark(toCompile: BenchmarkProject, zincEnabled: Bool
         Map.empty,
         outputToJar = false,
         subproject,
+        scalacOptions = nowarn,
```

Comment:
Could you move it to below the comment plz? I am guessing that the `List()` was passing into scalacOptions?